clang/lib/Headers/__clang_spirv_builtins.h

```diff
@@ -115,16 +85,6 @@ __spirv_GenericCastToPtr_ToGlobal(__generic const void *p,
                                   int) __SPIRV_NOEXCEPT {
   return (__global const void *)p;
 }
-static __SPIRV_overloadable __SPIRV_inline __global volatile void *
```

Comment:
Why removing these ? No binding it required as it boils down to an `addrspacecast` insn.